ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -498,3 +498,17 @@ def get_all_storageclass():
         )
     ]
     return storageclass
+
+
+def change_noobaa_endpoints_count(nb_eps):
+    """
+    Scale up or down the number of maximum NooBaa emdpoints
+
+    Args:
+        nb_eps (int): The number of required Noobaa endpoints
+
+    """
+    log.info(f"Scaling up Noobaa endpoints to a maximum of {nb_eps}")
+    params = f'{{"spec":{{"endpoints":{{"maxCount":{nb_eps},"minCount":1}}}}}}'
```

Comment:
If `minCount` is set to 1 then wouldn't the number of endpoints vary during testing (it might be noticeable when set e.g. to `5`)? Is this expected behaviour for setting `noobaa_endpoints` options?

Reply:
yes, the number could vary during testing. We will do an experiment of triggering the scale up of endpoints with loading the endpoint and see what happens during the tiers executions.

Reply:
Ok